app/test_main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import pytest
+from unittest.mock import patch
+
+from _pytest.monkeypatch import MonkeyPatch
+
+from app.main import can_access_google_page
+
+
+url = "https://google.com"
+
+
+@pytest.mark.parametrize(
+    "valid_url, has_internet, can_access",
+    [
+        (True, 7, "Accessible"),
```

Comment:
I can't understand the logic of this test case. Why the value of has_internet is int?

Reply:
In the very begining I made mistake and put to the parameters valid/invalid time. Then fixed it, but, I don't know why, only for 3 last cases)